Test/NativeUnitTest/NativeUnitTest.cpp

```diff
@@ -12,12 +12,9 @@ namespace NativeUnitTestForMain
 		
```

Comment:
#include "../Test/Source.cpp"
It is technically possible thing to include cpp. But you will not find any project which do this. It can raise a lot of problems in the future. The rule is:
- include only *.h files which contain declarations
- keep all definitions in *.cpp which are not included anywhere

Reply:
Will change in a later PR, just so I don't have to jump back and forth between them